example/src/main/java/com/attentive/example/ExampleApp.java

```diff
@@ -9,9 +9,9 @@
 
 public class ExampleApp extends Application {
     // Change this to your Attentive Domain to test with your Attentive account
-    private static final String ATTENTIVE_DOMAIN = "YOUR_ATTENTIVE_DOMAIN";
+    private static final String ATTENTIVE_DOMAIN = "games";
```

Comment:
I don't think you meant to commit this change (though if there was a way to parameterize this domain into some sort of config file we could gitignore that would be handy)

Reply:
You are right, I'll add that change since it's a small one and leave the domain in local.properties so setting up the project works out of the box without having to remember to change this.

Reply:
Also I'll update the local.properties file in 1Password to reflect the change for future developers.